app/code/Magento/Catalog/Helper/Image.php

```diff
@@ -787,7 +776,7 @@ protected function getImageFile()
      */
     protected function parseSize($string)
     {
-        $size = explode('x', strtolower($string));
+        $size = $string ? explode('x', strtolower($string)) : [''];
```

Comment:
Hello @karyna-tsymbal-atwix 
I think we need to check `$string` on `null`, also we can use empty array here `[]` 

```suggestion
        $size = $string !== null ? explode('x', strtolower($string)) : [];
```

Reply:
you're right, fixed